MinimOsd_Extra/protocols/NMEA.h

```diff
@@ -150,23 +151,22 @@ bool parse_NMEA_char(char c, char *string) {
         switch(param) { // words in sentence
         case 0:    //frame identification
             frame = 0;
-            if (string[0] == 'G' && string[2] == 'G' && string[3] == 'G' && string[4] == 'A') frame = FRAME_GGA;
-            if (string[0] == 'G' && string[2] == 'R' && string[3] == 'M' && string[4] == 'C') frame = FRAME_RMC;
+            if (string[0] == 'G' && string[1] == 'P' && string[2] == 'G' && string[3] == 'G' && string[4] == 'A') frame = FRAME_GGA;
+            if (string[0] == 'G' && string[1] == 'P' && string[2] == 'R' && string[3] == 'M' && string[4] == 'C') frame = FRAME_RMC;
```

Comment:
No, I do not listen about and do not take a care about NMEAv4, sorry 
about that. I take care about standard and compatible versions only. So, 
MY bug is already fixed, thanks. I am not programmer and github 
enthusiast. I am only trying to make it work without enough knowledge.
(I did not mean any bad things when I wrote BUG! Please do not be angry 
about that.)